react/src/pages/EndpointListPage.tsx

```diff
@@ -153,9 +150,12 @@ const EndpointListPage: React.FC<PropsWithChildren> = ({ children }) => {
             type="text"
             icon={<SettingOutlined />}
             style={
-              isDestroyingStatus(row?.desired_session_count, row?.status) ||
-              (!!row.created_user_email &&
-                row.created_user_email !== currentUser.email)
+              isDestroyingStatus(
+                row?.desired_session_count ?? row?.replicas,
```

Comment:
```suggestion
                row?.replicas ?? row?.desired_session_count,
```
I think so it's better to consider replicas first. Please apply it below similar codes too.